shared/constants/config.tsx

```diff
@@ -74,6 +74,12 @@ export const urlToUsername = (url: {
   const username = usernameMatch.toLowerCase()
   return username
 }
+export const makeConfiguredAccount = I.Record<Types._ConfiguredAccount>({
+  fullname: '',
```

Comment:
i think we should keep user metadata in the user store and not scattered around. you can load it through this flow but the data (fullname) should be kept there